django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -75,15 +75,15 @@ Requires core.js, SelectBox.js and addevent.js.
             filter_input.id = field_id + '_input';
 
             selector_available.appendChild(from_box);
-            var choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', 'javascript:void(0);', 'id', field_id + '_add_all_link');
+            var choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', '#javascriptvoid', 'id', field_id + '_add_all_link');
```

Comment:
Yeah it needs to be some href otherwise the mouse pointer won't turn into a pointing hand. The other option is to use `<button/>`